command/sidechain/helper.go

```diff
@@ -33,15 +32,32 @@ func CheckIfDirectoryExist(dir string) error {
 	return nil
 }
 
-func GetAccountFromDir(dir string) (*wallet.Account, error) {
-	secretsManager, err := secretsHelper.SetupLocalSecretsManager(dir)
+func ValidateSecretFlags(dataDir, config string) error {
+	if config == "" {
+		if dataDir == "" {
+			return polybftsecrets.ErrInvalidParams
+		} else {
+			return CheckIfDirectoryExist(dataDir)
+		}
+	}
+
+	return nil
+}
+
+func GetAccount(dataDir, config string) (*wallet.Account, error) {
+	// get secret manager and allow reading from local directory (true at the end)
+	secretsManager, err := polybftsecrets.GetSecretsManager(dataDir, config, true)
 	if err != nil {
 		return nil, err
 	}
 
 	return wallet.NewAccountFromSecret(secretsManager)
 }
 
+func GetAccountFromDir(dir string) (*wallet.Account, error) {
+	return GetAccount(dir, "")
+}
```

Comment:
I don't see many benefits of this wrapper function, but it's your call whether you want to keep it.

Reply:
Will leave it for tests.